content/docs/use-cases/datasets-cache.md

```diff
@@ -0,0 +1,107 @@
+# Datasets Cache
+
+Datasets used in data science tend to exceed typical storage and computing
+capacities. Valuable time is wasted waiting for downloads, and the wait repeats
+across environments. Combined storage needs expand rapidly as more people and
+projects acquire the same data, creating duplication and increasing cost. Is
+there an effective way to perform this process?
+
+![](/img/storage-layers.png) _Data storage infrastructure_
+
+DVC's built-in data <abbr>caching</abbr> enables you to implement a simple and
+efficient storage layer globally — for all your team's projects. This approach
+can help in situations such as:
+
+- You need to speed up data transfers from a massive object store on the cloud,
+  or to move a growing data store out of your machine without slowing things
+  down. How do you plug in something in the middle (like a NAS drive) to
+  existing projects?
+
+- Upgrading your storage platform is expensive. Can you pay only for fast access
+  to the data sets that are needed regularly?
+
+- You want to de-duplicate files automatically when multiple people are working
+  on the same datasets (for example, on a
+  [shared development server](#example-shared-development-server)).
+
+- Your team shares access to a GPU server for machine learning
+  [experiments](/doc/user-guide/experiment-management). How can you switch the
+  working datasets quickly, without re-downloading every time?
+
+## How it works
+
+DVC converts your raw data, intermediate artifacts, and final results (e.g.
+machine learning models) into unique storage objects. This optimizes storage by
+avoiding file duplication. <abbr>Cached</abbr> files are mapped back to your
+<abbr>workspace</abbr> using [file links] to minimize transfers.
+
+> 📖 See [Dataset Optimization](/doc/user-guide/large-dataset-optimization) for
+> more details.
+
+[file links]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+
+To unify data storage across multiple projects, you can configure a [shared
+cache]. This compounds the deduplication and performance benefits. Having
+separated data storage from other aspects of your projects, you can treat it as
+an independent infrastructure layer, using Ceph,
+[EFS](https://aws.amazon.com/efs/), object storage (e.g. S3,
+[GCS](https://cloud.google.com/storage/), Azure), or any other platform.
```

Comment:
Sure, it's possible to use object storage for your shared cache, but is this really typical? It seems we are mixing the same technologies between the cache and remote concepts, which makes it hard for me to understand the suggested patterns. What do you think about moving object storage (and the example types) down to the next paragraph where remotes are mentioned?

Reply:
> it's possible to use object storage for your shared cache, but is this really typical?
moving object storage (and the example types) down

Good point. I guess it wouldn't bring it closer. Moved ⬇️ 

> we are mixing the same technologies between the cache and remote concepts

This is the case though. There's no clear line.